src/quartz_api/internal/backends/dummydb/client.py

```diff
@@ -179,6 +179,36 @@ async def post_site_generation(
     ) -> None:
         pass
 
+    @override
+    async def get_substations(
+        self,
+        authdata: dict[str, str],
+    ) -> list[internal.Site]:
+        uuid = str(uuid4())
+
+        site = internal.Site(
+            site_uuid=uuid,
+            client_site_id=1,
+            latitude=26,
+            longitude=76,
+            capacity_kw=76,
+        )
+
+        return [site]
+
```

Comment:
Do we want the `get_substation_forecast` method in here as well?

Reply:
Ah - I in fact intended to remove that route to swap it to _get_predicted_solar_generation_for_location! Good spot

Reply:
No, we don't need it, the `get_substation_forecast` _route_  uses the `get_predicted_solar_power_production_for_location` database client method under the hood, so there's nothing new to override for this implementation of the client (it already has the `get_predicted_solar_power_production_for_location` method).

However, I might have to change this now we know the logic needs to be a bit different!